# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

VelocityCommandActionGoal action_goal
VelocityCommandActionResult action_result
VelocityCommandActionFeedback action_feedback
